src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReportData.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.reports.hearingsbyhearingtype;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Getter;
+import uk.gov.hmcts.ecm.common.model.listing.ListingData;
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+public class HearingsByHearingTypeReportData extends ListingData {
+
+    @JsonIgnore
+    private final HearingsByHearingTypeReportSummaryHdr reportSummaryHdr;
+    @JsonIgnore
+    private final List<HearingsByHearingTypeReportSummary> reportSummaryList = new ArrayList<>();
+    @JsonIgnore
+    private final List<HearingsByHearingTypeReportSummary2Hdr> reportSummary2HdrList = new ArrayList<>();
+    @JsonIgnore
+    private final List<HearingsByHearingTypeReportSummary2> reportSummary2List = new ArrayList<>();
+    @JsonIgnore
+    private final List<HearingsByHearingTypeReportDetail> reportDetails = new ArrayList<>();
+
+    public HearingsByHearingTypeReportData(HearingsByHearingTypeReportSummaryHdr reportSummaryHdr) {
+        this.reportSummaryHdr = reportSummaryHdr;
+    }
+
+    public HearingsByHearingTypeReportSummaryHdr getReportSummaryHdr() {
+        return reportSummaryHdr;
```

Comment:
Just a note: because you have added @Getter annotation at class level, the public getters for private fields are redundant as the framework add them for you.